### ISSUE TITLE
vmui: Uncaught RangeError: Invalid time zone specified: Etc/Unknown

### ISSUE THREAD

**Issue by valyala** [VictoriaMetrics/VictoriaMetrics#5732]
### Describe the bug

vmui stops loading with the following error after changing the timezone settings in Ubuntu 22.04 to automatic detection:

```
Uncaught RangeError: Invalid time zone specified: Etc/Unknown
    at Date.toLocaleString (<anonymous>)
    at u.tz (main.5c627264.js:2:13816)
    at Sn.map.ft.until (main.5c627264.js:2:143616)
    at An (main.5c627264.js:2:143940)
    at main.5c627264.js:2:144523
    at main.5c627264.js:2:528346
    at main.5c627264.js:2:528349
```

### To Reproduce

1. Start Ubuntu 22.04.
2. Go to setting -> date/time and enable `automatic timezone` checkbox there.
3. Restart the computer.
4. Open Chrome browser.
5. Navigate to https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/?

`vmui` doesn't open after that - it shows blank window. When pressing `F12`, Chrome shows the error above.

I tried investigating it - it looks like the issue is in the `dayjs` third-party dependency - it incorrectly detects browser timezone as `Etc/Unknown`, which triggers the error above when this timezone is passed to `(new Date()).toLocaleString("en-US", { timeZone: "Etc/Unknown" })`.

Probably, the best solution would be do stop using `dayjs` dependency and to write the needed code on our side.

### Version

The latest commit in `master` branch

**Comment by Loori-R** [comment]
This is not a problem with `dayjs`, defining the timezone using standard JavaScript functions can also return `Etc/Unknown`.

`Etc/Unknown` usually means that the specified timezone is not recognized or supported, or the timezone information cannot be determined based on user preferences.

I will add handling for this case and display a message to the user.

**Comment by valyala** [comment]
This issue has been fixed in [v1.98.0](https://docs.victoriametrics.com/CHANGELOG.html)

**Comment by ahmedhosnypro** [comment]
# RangeError: Invalid Time Zone Specified (Etc/Unknown) - JavaScript Timezone Detection Failure

## Problem
This error occurs when JavaScript fails to automatically detect the browser's timezone.

## Error Message
```
RangeError: Invalid time zone specified: Etc/Unknown
    at Date.toLocaleString (<anonymous>)
```

## Common Causes
- Misconfigured system timezone
- Incomplete browser timezone database
- Virtual machine or containerized environment limitations

## Solution

### System Timezone Configuration
```bash
sudo ln -sf /usr/share/zoneinfo/Africa/Cairo /etc/localtime
```

### Verify Current Timezone
```bash
date
timedatectl
```

## Timezone Examples
- `Africa/Cairo`
- `Asia/Riyadh`
- `Europe/London`
- `America/New_York`
- `UTC`

## JavaScript Debugging
```javascript
// Detect and log current timezone
try {
  const userTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
  console.log('Detected Timezone:', userTimezone);
} catch (error) {
  console.warn('Timezone detection failed');
}
```

## Recommended Steps
1. Check system timezone settings
2. Ensure browser's regional settings are correct
3. Use a timezone selection dropdown in your application
4. Consider using robust timezone libraries like `moment-timezone`
